src/equations/ideal_glm_mhd_2d.jl

```diff
@@ -1657,7 +1659,9 @@ end
 
 # Transformation from conservative variables u to d(p)/d(u)
 @inline function gradient_conservative(::typeof(pressure),
-                                       u, equations::IdealGlmMhdEquations2D)
+                                       u,
+                                       equations::Union{IdealGlmMhdEquations2D,
+                                                        IdealGlmMhdEquations3D})
```

Comment:
I realized that this function was not present for 3D MHD, and decided to unify it with the 2D function. Do we want to keep it like this or have repeated code for 2D and 3D?

Reply:
I had some troubles with a unified version of `is_valid` while precompiling Trixi for the first time (`IdealGlmMhdEquations3D` wasn't defined while precompiling this file). Do you also had this?
If so, we need a repeated version